tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -764,7 +764,7 @@ private function assertTypeValidity(ReflectionType $reflected, string $specified
         $callMapType = Type::parseString($specified);
 
         try {
-            $this->assertTrue(UnionTypeComparator::isContainedBy(self::$codebase, $callMapType, $expectedType), "{$msgPrefix} type '{$specified}' should be contained by reflected type '{$reflected}'");
+            $this->assertTrue(UnionTypeComparator::isContainedBy(self::$codebase, $callMapType, $expectedType, allow_float_int_equality: false), "{$msgPrefix} type '{$specified}' is not contained by reflected type '{$reflected}'");
```

Comment:
This only affected one function. We want an explicit check for callmap validity.